src/tests/multihost/ad/test_samba_data.py

```diff
@@ -14,7 +19,9 @@ class Testsmbsecretrotation(object):
     @pytest.mark.tier2
     def test_0001_rotation(self, multihost):
         """
-        @Title: Verify machine passwd updates local smb secrets
+        :title: Verify machine passwd updates local smb secrets
+        :id: 3d08ea1c-6724-4bc9-ac62-b8be66486ee4
+        :requirement: Add support for passing --add-samba-data to adcli
```

Comment:
I think requirement parameter should be part of script not part of function :
""
casecomponent: sssd
:subsystemteam: sst_identity_management
:upstream: yes
:requirement: samba, adcli 
"""

Reply:
Moved it. Requirement string should match the requirement in polarion to be automatically linked to the test case.